packages/common/test/content/computeProps.test.ts

```diff
@@ -0,0 +1,106 @@
+import {
+  computeProps,
+  deriveLocationFromItemExtent,
+  getItemExtent,
+} from "../../src/content/_internal/computeProps";
+import { MOCK_AUTH } from "../mocks/mock-auth";
+import { ArcGISContextManager } from "../../src/ArcGISContextManager";
+import { IPortal } from "@esri/arcgis-rest-portal";
+import { IUser } from "@esri/arcgis-rest-types";
+import { IHubEditableContent, IModel } from "../../src";
+
+describe("content computeProps", () => {
+  let authdCtxMgr: ArcGISContextManager;
+  beforeEach(async () => {
+    // When we pass in all this information, the context
+    // manager will not try to fetch anything, so no need
+    // to mock those calls
+    authdCtxMgr = await ArcGISContextManager.create({
+      authentication: MOCK_AUTH,
+      currentUser: {
+        username: "casey",
+        privileges: ["portal:user:createItem"],
+      } as unknown as IUser,
+      portal: {
+        name: "DC R&D Center",
+        id: "BRXFAKE",
+        urlKey: "fake-org",
+        properties: {
+          hub: {
+            enabled: true,
+          },
+        },
+      } as unknown as IPortal,
+      portalUrl: "https://org.maps.arcgis.com",
+    });
+  });
+  it("getItemExtent isBBox is true", () => {
```

Comment:
you should move this into it's own describe - you don't need the context manager that is set up in the `befoeEach()`